docs/feature-flagging/targeting.md

```diff
@@ -1,7 +1,3 @@
----
-sidebar_position: 2
```

Comment:
are we defining the sidebar_position somewhere else now? I figure targeting should be one of the first ones on the side panel